legacyman_parser/parser.py

```diff
@@ -49,6 +50,15 @@ def parse_from_root():
     if path_has_back_slash(arg_url) or path_has_drive_component(arg_url):
         cleansed_url = get_cleansed_path(arg_url)
 
+    # test_payload_for_parsed_json
+    kw_args = dict(arg.split('=') for arg in sys.argv[2:])
+    test_payload_json = None
```

Comment:
@rnllv - I think we can simplify the next two lines.  We check for the  presence of the keyword, and we try to extract it if it is present. I _guess_ we could just try to extract it, then check that `test_payload_json` is `not None` (like we do later on).

Reply:
Python will [throw](https://stackoverflow.com/a/10116540/11905887) a [KeyError](https://docs.python.org/3/library/exceptions.html#KeyError).

This is one aspect that Java handles better than Python :)

Diff:
![image](https://user-images.githubusercontent.com/77854924/208728028-07a59878-a17a-4c35-9f18-9e19f33868de.png)

Result:
![image](https://user-images.githubusercontent.com/77854924/208727739-44791997-1ee6-4457-9837-9a6fd4569836.png)

Reply:
Sure, ok. Thanks for the teach-in.